package.json

```diff
@@ -146,7 +148,7 @@
         "less": "^3.11.3",
         "mocha": "^7.2.0",
         "node-loader": "^2.0.0",
-        "npm": "^6.14.5",
```

Comment:
No idea why `npm` was in package.json, removed this, definitely not required (must have been accidentaly installed without the `-g` flag)